modern/src/App.js

```diff
@@ -21,36 +22,59 @@ async function getSkin() {
   return await initialize(zip, skinXml);
 }
 
+function handleMouseEventDispatch(node, event, eventName) {
+  const rect = event.target.getBoundingClientRect();
```

Comment:
I just want to call out that this is likely very expensive (it forces a layout). We will probably need to find a way to avoid this call when we don't need it.